src/mlpack/methods/reinforcement_learning/environment/ftn_param.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file methods/reinforcement_learning/environment/ftn_param.hpp
+ * @author Nanubala Gnana Sai
+ *
+ * This file holds the parameters of the Fruit Tree Navigation (FTN) Task:
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_RL_ENVIRONMENT_FTN_PARAM_HPP
+#define MLPACK_METHODS_RL_ENVIRONMENT_FTN_PARAM_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+
+namespace rl {
+
+      /**
+       *  Maps the depth value of a Fruit Tree to it's corresponding Convex Convergence Set (CCS). 
+       *  The set of reward vectors R^6 at the leaf nodes are pre-defined to   
+       */
+      static const std::unordered_map<size_t, arma::mat> ConvexSetMap =
+      {
+        {
+          '5', arma::mat(
+              {
+                {3.67917966,7.49190652,3.14072363,0.03085158,4.02109647,2.96104264,3.95849765,0.74185053,4.28506840,5.93102382,1.05416268,0.79064992,4.35145246,2.44514113,6.66924742,4.45750971,1.70141153,1.61886242,2.36047279,0.52087376,4.85800708,3.51780254,0.92487495,0.91807423,1.00725144,1.89889027,4.57737545,1.23346934,1.51947586,3.81593770,5.85626898,4.24912001}, 
```

Comment:
```suggestion
{3.67917966,7.49190652,3.14072363,0.03085158,4.02109647,2.96104264,3.95849765,0.74185053,4.28506840,5.93102382,1.05416268,0.79064992,4.35145246,2.44514113,6.66924742,4.45750971,1.70141153,1.61886242,2.36047279,0.52087376,4.85800708,3.51780254,0.92487495,0.91807423,1.00725144,1.89889027,4.57737545,1.23346934,1.51947586,3.81593770,5.85626898,4.24912001}, 
```

Remove the extra empty line here.